### PR TITLE
Issue 3 - Set up front end with file structure

### PR DESCRIPTION
**Pull request by lhuijser**
Changes:

- Ran create-react-app with react version 16 rather than the latest
- Set up base file structure within the frontend for styling, components, and pages

Related issue:
#3

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.10",
+    "@testing-library/react": "^11.2.6",
+    "@testing-library/user-event": "^12.8.3",
+    "react": "^16.0.0",
```
Comment (Denijar) [inline frontend/package.json:9]: sorry I pranked us. Might be easier if we use the same libraries that Andrew is using for assignment 1. Should just be a case of editing this file to include the following:
```
    "@material-ui/core": "^4.11.3",
    "@material-ui/icons": "^4.11.2",
    "@material-ui/pickers": "^3.3.10",
    "axios": "^0.21.1",
```
(under dependencies)
and
```
    "@types/jest": "^26.0.22",
    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.0",
    "enzyme": "^3.11.0",
    "jest-enzyme": "^7.1.2",
    "jest-mock-axios": "^4.3.0",
    "react-test-renderer": "^17.0.2"
```
(under devDependencies)

**Review by Denijar — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 1 above)
Comment (Denijar) [reply frontend/package.json:9]: There are a bunch of other dependencies that he has (i.e. workbox) but maybe we should only install those if/when we know we are going to use them

Comment (lhuijser) [reply frontend/package.json]: Done! I have added these and restored the original react version :) thanks for this!

**Review by Denijar — APPROVED** [review]
<3

**Review by Denijar — APPROVED** [review]
<3 but this time actually after you push the commit lmao

**Pull request merged**